src/Ros_mpGetRobotCalibrationData.c

```diff
@@ -15,9 +18,140 @@ LONG Ros_mpGetRobotCalibrationData(ULONG file_no, MP_RB_CALIB_DATA *rData)
     return mpGetRobotCalibrationData(file_no, rData);
 
 #elif defined (DX200) || defined (FS100)
-    #error Not implemented (yet)
+    //The download and parsing of the DAT file is expensive. I only want to
+    //perform it once. So let's cache the data in an array.
+    //
+    //This is an array of POINTERS. We'll only allocate a slot if that calibration
+    //file is configured.
+    static MP_RB_CALIB_DATA* calibrationFiles[MAX_ROBOT_CALIBRATION_FILES];
+    
+    if (file_no == 0) //first time in, let's parse the whole file
```

Comment:
Another question: can the calibration be updated without restarting the controller?

If so, the current implementation would make it impossible to (re)load updated data.